packages/next-swc/crates/next-core/src/next_server/context.rs

```diff
@@ -175,7 +179,8 @@ fn defines(mode: NextMode) -> CompileTimeDefines {
         process.turbopack = true,
         process.env.NODE_ENV = mode.node_env(),
         process.env.__NEXT_CLIENT_ROUTER_FILTER_ENABLED = false,
-        process.env.NEXT_RUNTIME = "nodejs"
+        process.env.NEXT_RUNTIME = "nodejs",
+        process.env.__NEXT_EXPERIMENTAL_REACT = false,
```

Comment:
Does this need to be set to true for server actions to function? I'll need to update this in my actions branch.

Reply:
yup

Reply:
I think we'll need to add yet another compiler for turbo + experimental react